### DISCUSSION TITLE
Patching Director Certificate failed

### DISCUSSION THREAD

**Discussion by mstohr1975** [garrynewman/Control4.Jailbreak#17] [Q&A]
Hi there, 

I tried to patch 3.2.3.606460-res director and failed with error message: 

"Copying To Director
Connecting to director via SCP..  .. connected!
Downloading /etc/openvpn/clientca-prod.pem
Done - got 11049 bytes

Uploading /etc/openvpn/clientca-prod.2021-25-9--16-33-42.backup

scp: error: unexpected filename: 
   bei Renci.SshNet.ScpClient.CheckReturnCode(Stream input)
   bei Renci.SshNet.ScpClient.UploadFileModeAndName(IChannelSession channel, Stream input, Int64 fileSize, String serverFileName)
   bei Renci.SshNet.ScpClient.Upload(Stream source, String path)
   bei Garry.Control4.Jailbreak.DirectorPatch.PatchDirector(LogWindow log) in C:\git\Control4.Jailbreak\UI\DirectorPatch.cs:Zeile 66.
   bei Garry.Control4.Jailbreak.DirectorPatch.PatchDirectorCertificates(Object sender, EventArgs e) in C:\git\Control4.Jailbreak\UI\DirectorPatch.cs:Zeile 146."

Now trying to connect with Composer HE or Pro I get the following error. Restarting of director did not work:
"Could not authenticate the connection to Director at 192.... Common causes are incorrect time/date settings on the controller or on the computer running Composer". I validated the system times, there max. 20-30 seconds difference.

Any ideas?

**Comment by OneScuzzie2k** [comment]
I have recomplied the executable with a new version of SSH and its working. I had the same issue.

**Reply by diegoahernandez** [reply]
can you share this new executable please? I'm having issues logging in!

**Reply by mstohr1975** [reply]
Anybody who could help with the updated executables to work with 3.2.3, please? Much appreciated.

**Reply by B08Z** [reply]
Any update

**Reply by cgbale** [reply]
This issue was resolved with the latest release quite some time ago. 

If you are having issues connecting to a new/blank/defaulted controller that has not been activated, use the new controller default password to connect instead.

**Reply by B08Z** [reply]
Thanks for the reply. 
The controller has been activated is the issue. I am running the control software in a VM and neither the default password or leaving it blank work. 

Not sure if there is a fix. 

I have added a user to the controller but I get an error related to permission denied when copying the director certs over. 

Any ideas. Sorry this might be a little disjointed.
